milestone.md

```diff
@@ -0,0 +1,199 @@
+Kamień milowy do: 6.12.2022r
+
+# Założenia
+- Drużyny jednoosobowe, czyli każdy gra przeciw każdemu
+
+## Stany gry
+Gra może znajdować się w jednym z następujących stanów:
+- przygotowanie (domyślny przy tworzeniu)
+- oczekiwanie na graczy
+- rozgrywka
+- zakończona
+
+Na podstawie stanu decydowane jest jakie akcje w endpointach mogą być wykonywane, a jakie nie. 
+
+Wpływ stanów na uprawnienia:
+- przygotowanie
+
+  GM tworzy i edytuje zadania
+
+  Gracz nie widzi gry na liście
+
+- oczekiwanie na graczy
+
+  Gracz widzi grę i jej opis
```

Comment:
Czy ma możliwość utworzenia drużyny/dołączenia, czy następuje to dopiero po starcie?

Reply:
Tak, na tym etapie odbywa się dołączanie i tworzenie drużyn